screwdriver.yaml

```diff
@@ -48,7 +48,7 @@ jobs:
             screwdriver.cd/dockerCpu: HIGH
             screwdriver.cd/dockerRam: HIGH
         requires: publish
-        template: sd/dind@latest
```

Comment:
Temporarily specifying the version instead of the tag as OSSD pipeline to build and deploy screwdriver is broken.